### PR TITLE
Dockerfile for testing on alpine linux

### PR DESCRIPTION
**Pull request by jesseduffield**
Just for the sake of easier testing on linux. We can build upon this as we go along

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# run with:
+# docker build -t lazygit
```
Comment (dawidd6) [inline Dockerfile]: this command lacks a directory, a single dot should be added at the end if we assume one is in lazygit directory already with Dockerfile in it, so:
`docker build -t lazygit .`

**Comment by dawidd6** [comment]
Maybe we should add:
`CMD ["lazygit"]`
at the end of Dockerfile, so after we `docker run ...` lazygit will open automatically.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# docker run -it lazygit:latest
+
+FROM golang:alpine
+
+RUN apk add git
+RUN apk add xdg-utils
```
Comment (dawidd6) [inline Dockerfile]: tbh i would squash these two `RUN`s to one of this form:
`RUN apk add -U git xdg-utils`
so we can have one layer with all dependencies needed

the `-U` flag is for apk's cache updating

Comment (jesseduffield) [reply Dockerfile]: updated :)

**Comment by jesseduffield** [comment]
@dawidd6 I've updated based on your comments

I've left the dockerfile without a CMD or ENTRYPOINT because once you close lazygit the session will end, and also because I think it's likely that you'd want to do some setup e.g. running a repo generator script before opening the app

**Pull request merged**